src/scenes/home/press/pressLinks/pressLinks.js

```diff
@@ -0,0 +1,323 @@
+        <li>
+          <ReactGA.OutboundLink
+            eventLabel="myLabel"
+            to="https://www.blogs.va.gov/VAntage/33028/upskill-veterans-training-technology-jobs/"
+            target="_blank"
+          >
+            How to “upskill” Veterans’ training for technology jobs
+          </ReactGA.OutboundLink>
+        </li>
+        <li>
+          <ReactGA.OutboundLink
+            eventLabel="myLabel"
+            to="https://www.instagram.com/p/BbZT3Q-BBDC/"
+            target="_blank"
+          >
+            Oregon Veterans News Magazine
+          </ReactGA.OutboundLink>
+        </li>
+        <li>
+          <ReactGA.OutboundLink
+            eventLabel="myLabel"
+            to="https://blog.sabio.la/jameel-from-jarhead-to-softwareengineer-9130702da5c1"
+            target="_blank"
+          >
+            Jameel: From Marines to SoftwareEngineer
+          </ReactGA.OutboundLink>
+        </li>
+        <li>
+          <ReactGA.OutboundLink
+            eventLabel="myLabel"
+            to="https://www.seattletimes.com/business/technology/seattles-code-fellows-wins-approval-under-gi-bill/"
+            target="_blank"
+          >
+            Seattle coding-school tuition to be covered by GI Bill
+          </ReactGA.OutboundLink>
+        </li>
+        <li>
+          <ReactGA.OutboundLink
+            eventLabel="myLabel"
+            to="https://www.koin.com/news/veteran-gi-bill-should-cover-code-school_20180208085618706/960235415"
+            target="_blank"
+          >
+            Veteran: GI Bill should cover code school
+          </ReactGA.OutboundLink>
+        </li>
+        <li>
+          <ReactGA.OutboundLink
+            eventLabel="myLabel"
+            to="http://patriotbootcamp.org/blog/operation-code"
+            target="_blank"
+          >
+            Patriot Boot Camp and Operation Code join forces to help military veterans become
+            technology entrepreneurs
+          </ReactGA.OutboundLink>
+        </li>
+        <li>
+          <ReactGA.OutboundLink
+            eventLabel="myLabel"
+            to="http://www.maswired.com/operation-code-wants-veterans-to-work-in-tech/"
+            target="_blank"
+          >
+            Operation Code wants veterans to work in tech
+          </ReactGA.OutboundLink>
+        </li>
+        <li>
+          <ReactGA.OutboundLink
+            eventLabel="myLabel"
+            to="http://diversemilitary.net/2017/06/27/coding-bootcamps-accepts-gi-bill/"
+            target="_blank"
+          >
+            Coding Bootcamps Accepts GI Bill
+          </ReactGA.OutboundLink>
+        </li>
+        <li>
+          <ReactGA.OutboundLink
+            eventLabel="myLabel"
+            to="https://news.clearancejobs.com/2017/07/17/operation-code-looks-help-veterans-land-careers/"
+            target="_blank"
+          >
+            Operation Code Looks to Help Veterans Land IT Careers
+          </ReactGA.OutboundLink>
+        </li>
```

Comment:
![selection_081](https://user-images.githubusercontent.com/8835055/44129374-e5f97f1a-a015-11e8-80ff-1a7724d43e8b.png)

The list-items could use some vertical space between them on mobile to provide visual separation between elements. I suggest adjusting the line-height for the all `<li>` elements, and adjusting the margin-top for adjacent sibling `<li>` elements:
```
.item {
  line-height: 1;
}

.item + .item {
  margin-top: 5px;
}
```

Due to the small width of mobile screens, for the `<ul>` element, the bullets can probably be removed, and the padding can be adjusted: 
```
.list {
    list-style: none;
    padding: 0 20px;
}
```